changelogs/8.15.asciidoc

```diff
@@ -0,0 +1,35 @@
+[[apm-release-notes-8.15]]
+== APM version 8.15
+* <<apm-release-notes-8.15.0>>
+
+[float]
+[[apm-release-notes-8.15.0]]
+=== APM version 8.15.0
+
+https://github.com/elastic/apm-server/compare/v8.14.1\...v8.15.0[View commits]
+
+[float]
+==== Bug fixes
+
+- Avoid data race due to reuse of `bytes.Buffer` in ES bulk requests {pull}13155[13155]
+- APM Server now relies on the Elasticsearch apm-data plugin's index templates, which reverts some unsafe uses of `flattened` field types {pull}12066[12066]
+- Add `error.id` to jaeger errors {pull}13196[13196]
+
+[float]
+==== Breaking Changes
+
+[float]
+==== Deprecations
+
+[float]
+==== Intake API Changes
+
+[float]
+==== Added
+
+- Add `elasticsearch.flushed.uncompressed.bytes` metric {pull}13155[13155]
+- APM Server now relies on the Elasticsearch apm-data plugin's index templates, removing the requirement to install the APM integration package {pull}12066[12066]
```

Comment:
Looks a duplicate of [line 15](https://github.com/elastic/apm-server/pull/13552/files#diff-6422fbaf1143429df6cf50366dbf51f7e970a9cad771b57e5a658acae518bfa3R15)? (looks more a new feature to me than a bug fix)

Reply:
I think L15 highlights: 
> reverts some unsafe uses of `flattened` field types

from the same commit.

Reply:
The PR is the same, but I think the same PR was categorized as both bug fix and addition?

Reply:
thanks for the clarification! Fine by me